guided-charts/aws-hyperpod/templates/authmiddleware/deployment.yaml

```diff
@@ -70,8 +70,10 @@ spec:
             value: "{{ .Values.authmiddleware.writeTimeout }}"
           - name: SHUTDOWN_TIMEOUT
             value: "{{ .Values.authmiddleware.shutdownTimeout }}"
-          - name: JWT_VALIDATION_TYPE
+          - name: JWT_SIGNING_TYPE
             value: "{{ .Values.authmiddleware.jwtValidationType }}"
+          - name: KMS_KEY_ID
```

Comment:
We'll need to revisit this post-GA; the OSS middleware cannot have vendor-specific keywords like KMS.
That can be a parameter of a plugin sidecar.